Tests/External/Plugins/ASCompletion.Tests/Completion/ASCompleteTests.cs

```diff
@@ -719,17 +719,23 @@ public IEnumerable<TestCaseData> TestCases
             [Test, TestCaseSource(nameof(TestCases))]
             public string AS3(string text)
             {
-                sci.ConfigurationLanguage = "as3";
-                ASContext.Context.SetAs3Features();
-                return Common(text, sci);
+                if (sci.ConfigurationLanguage != "as3")
```

Comment:
Nah I think it's fine. Those methods took around 300ms per run, and in practice, the context doesn't change this frequently and I believe `Set*Features()` aren't called this frequently either.